src/components/base/BaseModal.vue

```diff
@@ -58,6 +58,10 @@ export default {
 			type: String,
 			default: "medium",
 		},
+		onBackDrop: {
```

Comment:
```suggestion
		onBackdropClick: {
```
I don't like the naming of this prop, because it does not tell when this callback will be called, only that it has something to do with the backdrop.
- [ ] rename prop

Reply:
- [ ] Please add a test for this behaviour